src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  const stylesObject = styles.reduce((acc, style) => {
+    const [property, value] = style.split(':', 2)
+      .map((item) => item.trim());
```

Comment:
When splitting the `style` string, there might be more than one colon in the property value, for example in a font declaration like 'font: bold 12px/30px Georgia, serif;'. Using `split(':', 2)` will only capture 'bold' as the value, which is incorrect. Consider using a different approach to split the property and value that accounts for multiple colons in the value.